### DISCUSSION TITLE
LyCORIS moved to Lora folder (by me)

### DISCUSSION THREAD

**Discussion by Manniala** [BlafKing/sd-civitai-browser-plus#97] [Ideas]
Hey, i have sorted every Model, Lora... into folders, and  as "stupid" as i was, i did not think about that your Extension, would not be able to detect them in the Lora folder.

It was just easier to manage when there only were 1 set of folder, so i want to suggest, that your Extension could also search the Lora folder for LyCORIS models.

If not, thats ok, would be a lot of work for you, it was my mistake (yep i do a lot of those :D), and i than just have to find the LyCORIS and sort then out :)

**Comment by BlafKing** [comment]
If I understand you correctly, then this should already be implemented!
If you go to:
`Settings Tab > CivitAI Browser+` 
Then you should be able to find a setting called `Treat LoCon's as LORA's`, you can use that option to detect LyCoris models inside the LORA folder, since it will then treat them as LORA's :)

**Reply by Manniala** [reply]
Thx :D
![image](https://github.com/BlafKing/sd-civitai-browser-plus/assets/32381860/b0385021-e4c5-4f8a-9295-b5e94a080cbc)

**Reply by BlafKing** [reply]
No problem, and great image 😂
